client/my-sites/customer-home/main.tsx

```diff
@@ -28,13 +28,15 @@ export default function CustomerHome( { site }: { site: SiteDetails } ) {
 		retry: false,
 	} );
 
+	const shouldShowLaunchpadFirst = useShouldShowLaunchpadFirst( site );
+
 	return (
 		<Main wideLayout>
 			<PageViewTracker path="/home/:site" title={ translate( 'My Home' ) } />
 			<DocumentHead title={ translate( 'My Home' ) } />
 			{ site.options && (
 				<>
-					{ shouldShowLaunchpadFirst( site ) && ! isFullLaunchpadDismissed ? (
+					{ shouldShowLaunchpadFirst && ! isFullLaunchpadDismissed ? (
```

Comment:
The `shouldShowLaunchpadFirst` value has now become async right? I think we need to handle the case when `useShouldShowLaunchpadFirst()` is in a loading?